src/main/java/gui/ShapeDrawer.java

```diff
@@ -0,0 +1,61 @@
+package gui;
+
+import java.awt.*;
+import java.awt.geom.AffineTransform;
+
+public class ShapeDrawer {
+
+    public static void drawEdge(Graphics2D g2, int x1, int y1, int x2, int y2, int lineStrokeWidth) {
```

Comment:
Skoro stworzyłeś wcześniej te klasy dla edge ze współrzędnymi, to można by to tutaj wykorzystać. Dotyczy to też pozostałych metod w tej klasie.